packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1436,6 +1442,7 @@ export class ContainerRuntime
 					this.summarizerNode.createChild(
 						summarizeInternal,
 						id,
+						channelsTreeName,
```

Comment:
Every usage in this PR passes `channelsTreeName`. Why did you decide to take this as a parameter instead of hardcoding it?

Reply:
Also, this PR doesn't give DataStore implementation a chance to use something different, since this is not a parameter of the `createSummarizerNodeFn` callback but baked in by `DataStores.getCreateChildSummarizerNodeFn`